### PR TITLE
add Math.min to cap percent at 100%

### LINKED ISSUE #35: Percentage over 100 shows as empty.
When I pass in a percentage greater than 100, it fills in the bgColor, I thought it would just stay 'full'.

I have values that are bigger than 100, My work around is `percent > 100 ? 100 : precent`

### PR DESCRIPTION
**Pull request by karltaylor**
This fixes #35 😊

If the percentage is over 100%, the stroke offset will always be full.

### CODE REVIEW

File: src/circle.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export interface CircleState {
 
 const radius = 175;
 const diameter = Math.round(Math.PI * radius * 2);
-const getOffset = (val = 0) => Math.round((100 - val) / 100 * diameter);
+const getOffset = (val = 0) => Math.round((100 - Math.min(val, 100)) / 100 * diameter);
```
Comment (zzarcon) [inline src/circle.tsx]: Can we have a unit test to cover this case? :)

**Comment by zzarcon** [comment]
Thanks for this @karltaylor ! I added a small comment, let me know if you need any help

File: src/circle.tsx (hunk 1 above)
Comment (karltaylor) [reply src/circle.tsx]: 😬I'll give it a shot, two ticks

File: __tests__/index.tsx [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
+ */
+
 import * as React from 'react';
-import { shallow, render, mount } from 'enzyme';
+import { shallow, configure } from 'enzyme';
 import Circle, {CircleProps} from '../src';
 
+const Adapter = require("enzyme-adapter-react-16");
+
+configure({ adapter: new Adapter() });
```
Comment (karltaylor) [inline __tests__/index.tsx]: I couldn't get it running without these changes due to these bugs/errors [here](https://stackoverflow.com/questions/51554366/jest-securityerror-localstorage-is-not-available-for-opaque-origins) and [here](https://github.com/airbnb/enzyme/issues/1284#issuecomment-389522919)

**Comment by karltaylor** [comment]
@zzarcon this GTM? :)

**Comment by zzarcon** [comment]
@karltaylor thanks a lot for the fix! And sorry for the delay :)

**Pull request merged**